Publisher/nl/email-editor-drag-and-drop-templates.md

```diff
@@ -1,17 +1,34 @@
 
-Als je graag snel en makkelijk campagnes wilt maken, zijn Drag-and-drop templates perfect voor jou. Door het slepen van blokken kun je eenvoudig aangeven waar teksten, afbeeldingen en buttons moeten worden geplaatst. Je kunt een drag-and-drop template aanmaken door in de **[E-mail editor](https://ms.copernica.com/#/design)** te kiezen voor *Aanmaken -> Drag-and-drop template aanmaken*.
+Als je graag snel en makkelijk campagnes wilt maken, zijn Drag-and-drop templates perfect voor jou. Door het slepen van blokken kun je eenvoudig aangeven waar teksten, afbeeldingen en buttons moeten worden geplaatst. Je kunt een drag-and-drop template aanmaken door in de **[E-mail-editor](https://ms.copernica.com/#/design)** te kiezen voor *Aanmaken -> Drag-and-drop-template aanmaken*.
 
 De editor is onderverdeeld in twee delen, namelijk de instellingen en een voorbeeldweergave van je template. De instellingen zijn ook weer onderverdeeld in twee delen (uiterlijk en inhoud). 
 
 ## Uiterlijk
+Hier kun je globaal de instellingen opgeven hoe je e-mail eruit komt te zien. Deze optie is onderverdeeld in 5 subkoppen:  
 
+**Algemene instellingen**:  
+Hier kun je gegevens als de breedte, het lettertype, de achtergrondkleur en de regelafstand van je template instellen.  
+
+**Lijnen**:  
+Hier kun je specifieke instellingen voor elke 'Lijn' binnen je template. Een lijn kan bestaan uit meerdere elementen. In de meeste gevallen zal je e-mail bestaan uit één lijn. Het kan echter voorkomen dat je bijvoorbeeld verschillende achtergrondkleuren wilt gebruiken. In dit geval kun je een extra lijn toevoegen aan je template en hier je instellingen wijzigen.  
+
+**Hoofdingen**:  
+Hier kun je de stijling aanpassen van kopteksten (hoofdingen).  
+
+**Knop**:  
+Hier kun je de stijling aanpassen van knoppen (buttons).
+
+**Mobiel formatteren**:  
+Hier kun je specifieke instellingen opgeven voor als je e-mail op een mobiel wordt geopend.  
 
 ## Inhoud
-Naast de uiterlijke kenmerken van je template, wil je ook blokken toevoegen aan je template. 
+Hier kun je de daadwerkelijke content toevoegen aan je e-mailtemplate.  
 
 ### Structuren
-Hier kun je de layout bepalen van je template. Wil je een contentblok over de gehele breedte hebben of wil je meerdere contentblokken gebruiken naast elkaar.
+Onder _Structuren_ bepaal je hoe de vormgeving van een rij met blokken (container) eruit komt te zien. Zo kun je kiezen voor een container over de gehele breedte van je e-mail of voor meerdere containers op één regel. Binnen een container kun je vervolgens een blok plaatsen om content te plaatsen. Door het gebruik van meerdere containers naast elkaar is het bijvoorbeeld mogelijk om een afbeelding en een tekst op dezelfde hoogte te plaatsen in je e-mail. 
+
+Door op het structuur-element te klikken in je template, kun je specifieke instellingen opgeven. Hierbij kun je denken aan de breedte van iedere container binnen dit element of hoeveel padding er tussen beide containers is.
 
 ### Blokken
```

Comment:
Noemen we dit nou containers of niet? Ik vind blokken goed hoor, maar dan moeten we eea aanpassen.